doc/dev_guide/building_from_source.rst

```diff
@@ -5,279 +5,226 @@
+    *   For CentOS 8, say:
 
-   * For Mac OS X (instructions below are for OS X El Capitan):
+        ..  code-block:: console
 
-     If you're using Homebrew as your package manager, say:
+            $ yum install epel-release
+            $ curl -s https://packagecloud.io/install/repositories/packpack/backports/script.rpm.sh | sudo bash
+            $ yum install -y \
+              git perl gcc cmake make gcc-c++ libstdc++-static autoconf automake libtool \
+              python3-yaml python3-six python3-gevent
 
-     .. code-block:: console
+    *   For Mac OS X:
 
-         $ brew install cmake make autoconf binutils zlib \
-                autoconf automake libtool \
-                readline ncurses openssl libunwind-headers icu4c \
-                && pip install python-daemon \
-                msgpack-python pyyaml configargparse six gevent
+        Before you start please install default Xcode Tools by Apple:
 
-     .. NOTE::
+        ..  code-block:: console
 
-         You can not install `zlib-devel <https://www.zlib.net/>`_  package this way.
+            $ xcode-select --install
+            $ xcode-select -switch /Applications/Xcode.app/Contents/Developer
 
-     Alternatively, download Apple's default Xcode toolset:
+        Install brew using command from
+        `Homebrew repository instructions <https://github.com/Homebrew/inst>`_.
 
-     .. code-block:: console
+        After that run next script:
 
-         $ xcode-select --install
-         $ xcode-select -switch /Applications/Xcode.app/Contents/Developer
+        ..  code-block:: console
 
-   * For FreeBSD (instructions below are for FreeBSD 10.1+ release), say:
+            $ brew install autoconf automake libtool cmake
+            $ pip --user -r test-run/requirements.txt
 
-     .. code-block:: console
+        ..  NOTE::
 
-         $ pkg install -y sudo git cmake gmake gcc coreutils \
-               autoconf automake libtool \
-               readline ncurses openssl libunwind icu \
-               python27 py27-pip py27-setuptools py27-daemon \
-               py27-msgpack py27-yaml py27-argparse py27-six py27-gevent
+            Read how to manually build tarantool using external package managers
+            (Homebrew or MacPorts) for Mac OS on
+            `GitHub <https://github.com/tarantool/tarantool/blob/master/README.MacOSX>`_.
 
-   If some Python modules are not available in a repository,
-   it is best to set up the modules by getting a tarball and
-   doing the setup with ``python setup.py`` like this:
+    *   For FreeBSD (instructions below are for FreeBSD 10.4+ release
+        and FreeBSD 11 release), say:
 
-   .. code-block:: console
+        ..  code-block:: console
 
-       $ # On some machines, this initial command may be necessary:
-       $ wget https://bootstrap.pypa.io/ez_setup.py -O - | sudo python
+            $ pkg install git cmake gmake readline icu \
+              python27 py27-yaml py27-daemon py27-msgpack
```

Comment:
`py27-msgpack` is unnecessary (it is the submodule of test-run). We don't use python-daemon. I don't see `six` module.